### PR TITLE
HKG: Add FW and Remove from dashcamOnly for 2019 Elantra GT i30

### PR DESCRIPTION
**Pull request by sunnyhaibin**
Add firmware for the 2019 Hyundai Elantra GT i30. Remove the platform from the `dashcamOnly` flag.

Route ID:
- stock longitudinal: `734ef96182ddf940|2022-10-02--16-41-44`
- openpilot longitudinal:

Thanks to community Elantra GT i30 owner Breadjarmang.

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
@sshane [this comment](https://github.com/sunnyhaibin/openpilot-1/blob/3f11edbde408cfec6ac4aea8a81fce904c07c283/selfdrive/car/hyundai/values.py#L112) says it's the same as the Elantra. any reason why it has a different openpilot platform?

**Comment by sshane** [comment]
Not that I'm aware of, this was before my time. I'll have a look at merging them soon

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -34,7 +34,6 @@ def __init__(self, CP):
 
     # To determine the limit for your car, find the maximum value that the stock LKAS will request.
     # If the max stock LKAS request is <384, add your car to this list.
-    elif CP.carFingerprint in (CAR.GENESIS_G80, CAR.GENESIS_G90, CAR.ELANTRA, CAR.HYUNDAI_GENESIS, CAR.ELANTRA_GT_I30, CAR.IONIQ,
                                CAR.IONIQ_EV_LTD, CAR.SANTA_FE_PHEV_2022, CAR.SONATA_LF, CAR.KIA_FORTE, CAR.KIA_NIRO_PHEV,
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:37]:
```suggestion
    elif CP.carFingerprint in (CAR.GENESIS_G80, CAR.GENESIS_G90, CAR.ELANTRA, CAR.HYUNDAI_GENESIS, CAR.ELANTRA_GT_I30, CAR.IONIQ,
                               CAR.IONIQ_EV_LTD, CAR.SANTA_FE_PHEV_2022, CAR.SONATA_LF, CAR.KIA_FORTE, CAR.KIA_NIRO_PHEV,
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -34,7 +34,6 @@ def __init__(self, CP):
 
     # To determine the limit for your car, find the maximum value that the stock LKAS will request.
     # If the max stock LKAS request is <384, add your car to this list.
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:36]:
```suggestion
    # If the max stock LKAS request is <384, add your car to this list.
    elif CP.carFingerprint in (CAR.GENESIS_G80, CAR.GENESIS_G90, CAR.ELANTRA, CAR.HYUNDAI_GENESIS, CAR.ELANTRA_GT_I30, CAR.IONIQ,
```

**Comment by sshane** [comment]
Do you know what harness this user's car uses? I don't see it in any of our old databases

**Comment by sshane** [comment]
This is actually the same platform as the Ceed with the same harness. Should we tend to combine these cases into single platforms (with two CarInfos) @adeebshihadeh?

**Comment by adeebshihadeh** [comment]
> This is actually the same platform as the Ceed with the same harness. Should we tend to combine these cases into single platforms (with two CarInfos) @adeebshihadeh?
> 
> https://en.wikipedia.org/wiki/Hyundai_i30

Yes, if they're the same as far as openpilot is concerned.

**Comment by sshane** [comment]
So these two marketed cars (i30 and Elantra GT) have SCC with stop and go, whereas the Elantra 2019 does not. And the transmission signal is slightly different than the Ceed, so we can't combine it there. Going to keep it separate as it's relatively different enough

**Comment by sshane** [comment]
Let's test longitudinal, this car wasn't in legacy safety mode cars before

**Comment by sunnyhaibin** [comment]
> Let's test longitudinal first too, this car wasn't in legacy safety mode cars before

@sshane Requested the user to test and will report back when tested.

**Comment by sshane** [comment]
@sunnyhaibin any updates?

**Comment by sshane** [comment]
Feel free to remove from legacy safety mode if you confirm longitudinal works

**Pull request merged**